### ISSUE TITLE
[Bug]: Runtime dependency on pytest-cov

### ISSUE THREAD

**Issue by kroeschl** [lextudio/pysnmp#85]
### Expected behavior

pysnmp-lextudio should not depend on pytest-cov (or pytest) because we don't actually use those except for in development testing.

### Actual behavior

I end up with pytest and pytest-cov installed in my production environment when I depend on newer pysnmp-lextudio.

### Detailed steps

Just `pip install pysnmp-lextudio>=6.2.0` and observe the results:
```
$ pip install pysnmp-lextudio==6.2.0
Looking in indexes: https://pypi.ni.systems
Collecting ...
Installing collected packages: pluggy, packaging, iniconfig, coverage, pytest, pytest-cov, pysnmp-lextudio
  Attempting uninstall: pysnmp-lextudio
    Found existing installation: pysnmp-lextudio 6.1.2
    Uninstalling pysnmp-lextudio-6.1.2:
      Successfully uninstalled pysnmp-lextudio-6.1.2
Successfully installed coverage-7.6.0 iniconfig-2.0.0 packaging-24.1 pluggy-1.5.0 pysnmp-lextudio-6.2.0 pytest-8.2.2 pytest-cov-4.1.0
```

### Python package information

6.2.0 and later

### Operating system information

Debian 13

### Python information

3.9.13

**Comment by lextm** [comment]
Fixed in release 6.2.3, but you need to run `pip install pysnmp` to get it.

**Comment by kroeschl** [comment]
Thanks for the quick fix!
